.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - run: npm run test:unit
   integration:
     docker:
-      - image: cimg/node:16.11
+      - image: cimg/node:16.13.2-browsers
```

Comment:
Theoretically, we could make the build tests their own step, then we only have to use the browser image for that step, right?

Reply:
Just to understand you correctly: That would mean that we'd also need to change the chain of commands in the package.json (since atm the tests are executed automatically bei `npm run build`), right? Otherwise those would also execute with the the "lint" step for example.
I am not sure if I'd be a big fan of that though since imho when I run `npm run build` as a dev I want those tests to be executed automatically to see if the output is okay.

What I will definitely change though is to remove the "-browsers" from the steps that don't include the build command.